readme.md

```diff
@@ -19,9 +19,11 @@ const firstChunkStream = require('first-chunk-stream');
 
 // unicorn.txt => unicorn rainbow
 fs.createReadStream('unicorn.txt')
-	.pipe(firstChunkStream({chunkLength: 7}, function (chunk, enc, cb) {
-		this.push(chunk.toUpperCase());
-		cb();
+	.pipe(firstChunkStream({chunkLength: 7}, function (err, chunk, enc, cb) {
```

Comment:
What kind of error? We already require the user to check the data length to ensure it's big enough.